tests/test_categorical_embedding.py

```diff
@@ -205,55 +205,97 @@ def test_embedding_transformer(regression_data):
     )
 
 
-# import numpy as np
-# import pandas as pd
-# from sklearn.datasets import fetch_california_housing, fetch_covtype
-
-
-# def regression_data():
-#     dataset = fetch_california_housing(data_home="data", as_frame=True)
-#     df = dataset.frame.sample(5000)
-#     df["HouseAgeBin"] = pd.qcut(df["HouseAge"], q=4)
-#     df["HouseAgeBin"] = "age_" + df.HouseAgeBin.cat.codes.astype(str)
-#     test_idx = df.sample(int(0.2 * len(df)), random_state=42).index
-#     test = df[df.index.isin(test_idx)]
-#     train = df[~df.index.isin(test_idx)]
-#     return (train, test, dataset.target_names)
-
-
-# def classification_data():
-#     dataset = fetch_covtype(data_home="data")
-#     data = np.hstack([dataset.data, dataset.target.reshape(-1, 1)])[:10000, :]
-#     col_names = [f"feature_{i}" for i in range(data.shape[-1])]
-#     col_names[-1] = "target"
-#     data = pd.DataFrame(data, columns=col_names)
-#     data["feature_0_cat"] = pd.qcut(data["feature_0"], q=4)
-#     data["feature_0_cat"] = "feature_0_" + data.feature_0_cat.cat.codes.astype(str)
-#     test_idx = data.sample(int(0.2 * len(data)), random_state=42).index
-#     test = data[data.index.isin(test_idx)]
-#     train = data[~data.index.isin(test_idx)]
-#     return (train, test, ["target"])
-
+@pytest.mark.parametrize("multi_target", [True, False])
+@pytest.mark.parametrize(
+    "continuous_cols",
+    [
+        [
+            "AveRooms",
+            "AveBedrms",
+            "Population",
+            "AveOccup",
+            "Latitude",
+            "Longitude",
+        ],
+        [],
+    ],
+)
+@pytest.mark.parametrize("categorical_cols", [["HouseAgeBin"], []])
+@pytest.mark.parametrize("continuous_feature_transform", [None, "yeo-johnson"])
+@pytest.mark.parametrize("normalize_continuous_features", [True, False])
+@pytest.mark.parametrize("target_range", [True, False])
+@pytest.mark.parametrize(
```

Comment:
target_transform, target_range, custom_metrics, cutom_loss, custom_optimizer, all can be set to default values

Reply:
Fixed